pages/community/styleguide/aboutguide.md

```diff
@@ -0,0 +1,35 @@
+# AsyncAPI Style Guide
+
+## Introduction
+Welcome to the AsyncAPI Style Guide. This page serves as a guideline to contributors on how to write, structure, and format content/documentation.
+
+These guidelines help us in maintaining language, tone, and voice throughout the website. It also makes it easier for multiple contributors to work together on a single doc while ensuring consistency, accuracy, and clarity. 
+
+## Writing Style
+Below are some recommendations to consider when writing your pages.
+
+### Have your target audience in mind
+Before you begin planning and constructing your content, it is important to know who you are writing for and how the readers will benefit from your content.
+This will help you in strategizing and simplifying your writing process.
+
+### Research
+Before creating your first draft, it is beneficial to research similar resources and use them as references. Avoid plagiarism or copying and pasting from other websites unless the content already exists on our site.
+
+### Follow the writing principles
+When writing our content or documentation, we aim to achieve the following:
```

Comment:
```suggestion
When writing documentation, we aim to achieve the following:
```

1. When writing `content or documentation`? Uh.. what is the difference between content and documentation? 😜 This is actually repetitive so let's stick to documentation. 
2. We should be consistent and use either `docs` or `documentation` each time throughout this doc. Please update all references to one of those options. (which option is up to you, but I'm def partial to shorter `docs` if that helps at all)